src/main/scala/nl/knaw/dans/easy/stage/EasyStageDataset.scala

```diff
@@ -69,10 +71,10 @@ object EasyStageDataset {
       _ <- writeFoxml(sdoDir, foxmlContent)
       _ <- writePrsql(sdoDir, PRSQL.create())
       _ <- writeJsonCfg(sdoDir, jsonCfgContent)
-    } yield ()
+    } yield emdContent
   }
 
-  private def createFileAndFolderSdos(dir: File, parentSDO: String)(implicit s: Settings): Try[Unit] = {
+  private def createFileAndFolderSdos(dir: File, parentSDO: String)(implicit s: Settings, rights: AccessCategory): Try[Unit] = {
```

Comment:
@rvanheest 
I chose for implicit because it is an invariant for the recursion (unfold the hidden lines). Funny thing though, as a consequence I had to make setting more explicit at the initial call. Perhaps I should make it explicit for createFileSdo which doesn't pass it down any more.